### PR TITLE
VACMS-4103: Repairs broken redirects.

### PR DESCRIPTION
**Pull request by ndouglas**
## Description

Relates to #4103.  Created an update hook rather than just running the query live because I wanted to make sure it didn't break content builds.

## Testing done
this very pr

### Select Team for PR review

- [x] `Core Application Team`
- [ ] `Product Support Team`

### CODE REVIEW

**Review by acrollet — COMMENTED** [review]
This is a nit but I think [hook_post_update_NAME](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Extension%21module.api.php/function/hook_post_update_NAME/9.2.x) is a bit more appropriate. We have a [post_update file](https://github.com/department-of-veterans-affairs/va.gov-cms/blob/master/docroot/modules/custom/va_gov_db/va_gov_db.post_update.php) in va_gov_db.

File: docroot/modules/custom/va_gov_backend/va_gov_backend.install [hunk 1]
```diff
@@ -424,3 +424,16 @@ function va_gov_backend_update_8010() {
...
 }
+
+/**
+ * Strip trailing slashes from redirects.
+ */
+function va_gov_backend_update_8011() {
+  $connection = \Drupal::database();
+  $connection->update('redirect')
+    ->fields([
+      'redirect_source__path' => 'TRIM(TRAILING \'/\' FROM redirect_source__path)',
```
Comment (acrollet) [inline docroot/modules/custom/va_gov_backend/va_gov_backend.install:435]: nifty query!

**Comment by ndouglas** [comment]
>Executes an update which is intended to update data, like entities.

That's not a nit, you're absolutely right.  Honestly, I forgot about hook_post_update.  Thanks!

**Comment by acrollet** [comment]
> > Executes an update which is intended to update data, like entities.
> 
> That's not a nit, you're absolutely right. Honestly, I forgot about hook_post_update. Thanks!

learned it from @indytechcook !

**Comment by ndouglas** [comment]
- [Before Update, Selected Rows](https://gist.github.com/ndouglas/6aadd8cb6705e1f2057c46d4905c38b6)
- [Before Update, Specifically Affected Rows](https://gist.github.com/ndouglas/a42c10d1d412764e9756fd914304d838)
- [After Update, Selected Rows](https://gist.github.com/ndouglas/589ea3726578649bf6be4d69f030dad1)

**Review by acrollet — APPROVED** [review]
The code is concise, does the job and I like the use of the expression - really nice way to write a self-documenting query. I tested a few randomly selected redirects and they go where they're supposed to according to the table. Nice work!

**Pull request merged**